packages/renderer/src/lib/context/scanner.ts

```diff
@@ -26,8 +26,13 @@ import { CharCode } from '../../../../main/src/plugin/util/charCode';
 export const enum TokenType {
   LParen,
   RParen,
+  Eq,
```

Comment:
is there a reason why you removed these options from the code we imported ?

like NotEq is not here, as well Lt, LtEq, Gt, GtEq, EOF and RegexOp and RegexStr ?

Reply:
i'm just adding the support for the rules i actually use

Reply:
ok but why not let them here as they were already implemented ?

Reply:
ok i'll add everything